src/test/java/com/blogify/blogapi/integration/conf/TestUtils.java

```diff
@@ -19,6 +18,7 @@ public class TestUtils {
   public static final String CLIENT1_TOKEN = "client1_token";
 
   public static final String CLIENT2_TOKEN = "client2_token";
+  public static final String CLIENT3_TOKEN = "client3_token";
```

Comment:
look what i was doing with " @PostMapping("/posts/{postId}/reaction") "

the name i used